src/SPC/util/SPCConfigUtil.php

```diff
@@ -52,6 +52,15 @@ public function __construct(?BuilderBase $builder = null, array $options = [])
      */
     public function config(array $extensions = [], array $libraries = [], bool $include_suggest_ext = false, bool $include_suggest_lib = false): array
     {
+        $extra_exts = [];
+        foreach ($extensions as $ext) {
+            $extra_exts = array_merge($extra_exts, Config::getExt($ext, 'ext-suggests', []));
+        }
+        foreach ($extra_exts as $ext) {
+            if ($this->builder?->getExt($ext) && !in_array($ext, $extensions)) {
+                $extensions[] = $ext;
+            }
+        }
```

Comment:
`DependencyUtil::getExtsAndLibs` will automatically sort the suggested extensions. And is there any reason we need to add suggested extensions here?

Reply:
>bin/spc spc-config swoole,swoole-hook-pgsql

Extension::buildShared calls SpcConfigUtil::config([$ext->name]).

So it only passes `swoole` but `swoole-hook-pgsql` is enabled in the builder. This code turns ['swoole'] into ['swoole', 'swoole-hook-pgsql'] before DependencyUtil.

Reply:
That would not work because it would pull in definitions for symbols that should not be available (looking at you, postgresql, defining libc methods...)